debian/changelog

```diff
@@ -51,7 +51,7 @@ snapcraft (2.35) xenial; urgency=medium
   * many: account for python shebang args in rewrite
 
   [ Leo Arias ]
-  * typo: replace occured with occurred
+  * typo: replace occurred with occurred
```

Comment:
Careful, probably don't want to fix this one.

Reply:
hmm right thanks for notifying